Formula/corectl.rb

```diff
@@ -33,10 +34,7 @@ def install
 
     cd path do
       system "opam", "init", "--no-setup"
-      qcow_format_revision = build.head? ? "master" : "96db516d97b1c3ef2c7bccdac8fb6cfdcb667a04"
-      system "opam", "pin", "add", "qcow-format",
-        "https://github.com/mirage/ocaml-qcow.git##{qcow_format_revision}"
-      system "opam", "install", "uri", "qcow-format", "ocamlfind"
+      system "opam", "install", "uri", "ocamlfind", "qcow-format"
```

Comment:
things got merged upstream and started being distributed in regular ocam repos. actually that pin ended lately being problematic (see https://github.com/TheNewNormal/corectl/issues/90)...